res/css/views/settings/tabs/user/_GeneralUserSettingsTab.scss

```diff
@@ -22,6 +22,13 @@ limitations under the License.
     margin-top: 0;
 }
 
+// TODO: Make this selector less painful
```

Comment:
might as well do that whilst you're here, doesn't seem large enough to be done on its own

Reply:
It's not trivial enough to be included here imo